samples/productSearch/similarProducts.v1p3beta1.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function getSimilarProductsFile(
```

Comment:
Missing `START`|`END` region tags

Reply:
This has been resolved as part of https://github.com/googleapis/nodejs-vision/pull/196#